typescript/cli/src/config/chain.ts

```diff
@@ -57,7 +57,8 @@ export async function createChainConfig({
       const clientName = await provider.send('web3_clientVersion', []);
       const port = rpcUrl.split(':').slice(-1);
       const client = clientName.split('/')[0];
-      return `${client}${port}`;
+      const chainName = `${client}${port}`;
```

Comment:
I dont think the port will ever be letters
I did this so that anvil:8545 and anvil:8546 etc would not collide

Reply:
removed this entirely per our convo